src/main/java/org/gradle/profiler/studio/tools/StudioSandboxCreator.java

```diff
@@ -15,18 +16,19 @@
      * If sandboxDir is not specified only plugins and logs sandbox will be created, but config and system folder won't be.
      * In that case Android Studio will use configs from default folder, additionally it will write to default folder.
      */
-    public StudioSandbox createSandbox(Path sandboxDir) {
+    public StudioSandbox createSandbox(@Nullable Path sandboxDir) {
```

Comment:
This class has no state, shall we turn it into `static`?

Reply:
Sure, we can, I moved it to static